src/wutil.cpp

```diff
@@ -564,7 +564,7 @@ file_id_t file_id_t::file_id_from_stat(const struct stat *buf)
 #if defined(__APPLE__) || defined(__DragonFly__) || defined(__FreeBSD__) ||  defined(__OpenBSD__) || defined(__NetBSD__)
     result.generation = buf->st_gen;
 #else
-    result.generation = 0;
+    result.generation = NULL;
```

Comment:
On Linux there is may be no such thing as an inode generation.

NULL in C++ is explicitly defined as 0, so I considered it an aesthetic difference. I'm certainly not an old hand here so I defer to your wisdom. I put it there to be explicit with myself that buf->st_gen does not exist on Linux and that member is no good. 

Makes no difference to me if it's set to zero by NULL or 0 in the end.